aws-opentelemetry-distro/src/amazon/opentelemetry/distro/aws_opentelemetry_distro.py

```diff
@@ -117,5 +115,7 @@ def _configure(self, **kwargs):
             # Disable AWS Application Signals by default
             os.environ.setdefault(APPLICATION_SIGNALS_ENABLED_CONFIG, "false")
 
+        super(AwsOpenTelemetryDistro, self)._configure()
```

Comment:
Just wonder if there was a reason that the super() call was made before we set most (but not all) of the environment variables.

Reply:
>Just wonder if there was a reason that the super() call was made before we set most (but not all) of the environment variables.

It was a code merge issue. In beta version, we have super._configure() occurred after Agentic default setting.